modules/unsupported/geopkg/src/main/java/org/geotools/geopkg/GeoPackage.java

```diff
@@ -829,6 +826,8 @@ FeatureEntry createFeatureEntry(ResultSet rs) throws SQLException, IOException {
     }
 
     void addGeoPackageContentsEntry(Entry e) throws IOException {
+        final SimpleDateFormat DATE_FORMAT = new SimpleDateFormat("yyyy-mm-dd'T'HH:MM:ss.SSS'Z'");
+//            DATE_FORMAT.setTimeZone(TimeZone.getTimeZone("GMT"));
```

Comment:
Likewise for geopackage, does the spec say anything about dates? They might have to be in GMT